### PR TITLE
update footer github-icon visibility w/Yash Vardhan <yashvardhan513@g…

### PR DESCRIPTION
**Pull request by Grumpyyash**
…mail.com>

Signed-off-by: Yash vardhan <yashvardhan513@gmail.com>

**Add visibility to github-icon on hovering**

This PR fixes the issue of invisibility of github-icon in the footer when hovered over it.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Grumpyyash** [comment]
The issue 
WhatsApp Image 2021-01-19 at 4 12 32 PM

Resolving it 
WhatsApp Image 2021-01-19 at 4 12 36 PM

File: src/components/SocialLinks-Color/index.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const SocialLinksColor = () => {
                         href="https://github.com/layer5io"
                         target="_blank"
                         rel="noreferrer"
+                        className="github-anchor"
```
Comment (Tanuj22) [inline src/components/SocialLinks-Color/index.js:40]: Is this class being used?

Comment (Grumpyyash) [reply src/components/SocialLinks-Color/index.js:40]: No, not actually, I was planning to use it but it wasn't required in the end, I am removing it

Comment (Grumpyyash) [reply src/components/SocialLinks-Color/index.js]: I have removed it,sir!

File: src/components/SocialLinks-Color/sociallinkscolor.style.js [hunk 2]
```diff
@@ -8,18 +8,21 @@ const SocialLinksWrapper = styled.div`
         a {
             margin: 0 0.3rem;
             width: 2.5rem;
-            filter: grayscale(1) invert(0.2);
+            filter: grayscale(1) invert(0);
```
Comment (Nikhil-Ladha) [inline src/components/SocialLinks-Color/sociallinkscolor.style.js:11]: Why the change for all the links?

**Comment by Nikhil-Ladha** [comment]
Also, please resolve the conflicts.

**Comment by Nikhil-Ladha** [comment]
The best fix would be to move the classname `github` to the `a` tag from the `img` tag and then add the hover effect to the classname `github` separately after the default `a:hover`.

**Comment by Grumpyyash** [comment]
@Nikhil-Ladha, sir I have done the required changes and resolved conflicts, please review now

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
```
diff --git a/src/components/SocialLinks-Color/index.js b/src/components/SocialLinks-Color/index.js
index 83c22bf9..c26f3ffd 100644
--- a/src/components/SocialLinks-Color/index.js
+++ b/src/components/SocialLinks-Color/index.js
@@ -37,9 +37,9 @@ const SocialLinksColor = () => {
                         href="https://github.com/layer5io"
                         target="_blank"
                         rel="noreferrer"
+                        className="github"
                     >
                         <img
-                            className="github"
                             height="30px"
                             src={github_icon}
                             alt="github"
diff --git a/src/components/SocialLinks-Color/sociallinkscolor.style.js b/src/components/SocialLinks-Color/sociallinkscolor.style.js
index 23833d2d..120c093c 100644
--- a/src/components/SocialLinks-Color/sociallinkscolor.style.js
+++ b/src/components/SocialLinks-Color/sociallinkscolor.style.js
@@ -8,23 +8,19 @@ const SocialLinksWrapper = styled.div`
         a {
             margin: 0 0.3rem;
             width: 2.5rem;
-            filter: grayscale(1) invert(0);
+            filter: grayscale(1) invert(0.25);
 
             img {
                 width: 2rem;
-
             }
 
             &:hover {
-                filter: grayscale(0) invert(0.2);
-            }
-            .github {
-                filter: grayscale(0) invert(0.5);
-            }
-            .github:hover {
-                filter: grayscale(0) invert(1);
+                filter: grayscale(0) invert(0);
             }
         }
+        .github:hover {
+            filter: grayscale(0) invert(1);
+        }
 
 
         .mail_icon {
```
This should work well.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The `className="github"` attribute should be there for the `a` tag of `Github` icon and not for the `img`. Please update that.

**Comment by Nikhil-Ladha** [comment]
Could you simply revert all your changes and then apply the change I have mentioned above?

**Review by Nikhil-Ladha — APPROVED** [review]
Nice!
